packages/protvista-variation-adapter/src/protvista-variation-adapter.js

```diff
@@ -1,11 +1,54 @@
 import groupBy from "lodash-es/groupBy";
+import flow from 'lodash-es/flow';
+import flatten from 'lodash-es/flatten';
+
 import ProtvistaUniprotEntryAdapter from "protvista-uniprot-entry-adapter";
 import getColor from "./variantColour";
+import filters, { getFilter } from './filters';
+
+
 export default class ProtvistaVariationAdapter extends ProtvistaUniprotEntryAdapter {
   constructor() {
     super();
   }
 
+  static get observedAttributes() {
+    return ['activefilters'];
+  }
+
+  get isManaged() {
+    return true;
+  }
+
+  attributeChangedCallback(name, oldValue, newValue) {
+    if (oldValue !== newValue) {
+      if (name !== 'activefilters') {
+        return;
+      }
+      const { sequence, variants } = this._adaptedData;
+      newValue = newValue.trim();
+      if (!newValue) {
+        this._fireEvent('load', { payload: {sequence, variants: []} });
+        return;
+      }
+      const filteredVariants = newValue
+        .split(',')
+        .map(name => getFilter(name))
+        .map(f => f.applyFilter(variants))
+        .flat();
```

Comment:
I'm not 100% sure whether this requires another polyfill to be imported for IE/Edge...if so it would be great to find an alternative way.

Reply:
Done (using lodash version)